src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/impl/SurveySvcRestClientImpl.java

```diff
@@ -139,18 +141,16 @@ public SurveyDTO findSurvey(final UUID surveyId) {
     SurveyDTO survey = null;
 
     try {
-      log.debug(
-          "about to get to the Survey SVC with surveyId {} from {}",
-          surveyId,
-          uriComponents.toUri());
+      log.with("survey_id", surveyId)
+          .with("uri", uriComponents.toUri())
+          .debug("about to get to the Survey SVC");
       responseEntity =
           restTemplate.exchange(uriComponents.toUri(), HttpMethod.GET, httpEntity, String.class);
       survey = getSurveyDtoFromResponseEntity(responseEntity);
     } catch (HttpClientErrorException e) {
       if (e.getStatusCode() == HttpStatus.NOT_FOUND) {
         return null;
       }
-      log.error("Client error with status code = {}", e.getStatusCode(), e);
```

Comment:
why have we removed this?

Reply:
error gets thrown after anyway? Not sure what value this log adds

Reply:
error is rethrown